en/modify-tidb-configuration.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Modify TiDB cluster configuration
+summary: Learn how to modify the configuration of TiDB clusters deployed in Kubernetes.
+---
+
+# Modify TiDB cluster configuration
+
+For a TiDB cluster, you can [update the configuration of components](https://docs.pingcap.com/tidb/stable/dynamic-config/) online using SQL statements, including TiDB, TiKV, and PD, without restarting the cluster components. However, for TiDB clusters deployed in Kubernetes, after you upgrade or restart the cluster, the configurations of some components might be overwritten by the `TidbCluster` CR. This leads to invalid online configuration update.
```

Comment:
```suggestion
For a TiDB cluster, you can [update the configuration of components](https://docs.pingcap.com/tidb/stable/dynamic-config/) online using SQL statements, including TiDB, TiKV, and PD, without restarting the cluster components. However, for TiDB clusters deployed in Kubernetes, after you upgrade or restart the cluster, the configurations updated using SQL statements will be overwritten by those in the `TidbCluster` CR. This leads to the online configuration update being invalid.
```

Reply:
I see that we use both `online` and `dynamically` in documentation. How about trying to keep the word usage consistent? Which one is better in your opinion?

Reply:
I prefer `online`. TiDB documentation uses `online` almost across the [whole doc](https://docs.pingcap.com/tidb/stable/dynamic-config) except in the URL slug.